transport/http2_server.go

```diff
@@ -180,28 +191,30 @@ func newHTTP2Server(conn net.Conn, config *ServerConfig) (_ ServerTransport, err
 	}
 	var buf bytes.Buffer
 	t := &http2Server{
-		ctx:             context.Background(),
-		conn:            conn,
-		remoteAddr:      conn.RemoteAddr(),
-		localAddr:       conn.LocalAddr(),
-		authInfo:        config.AuthInfo,
-		framer:          framer,
-		hBuf:            &buf,
-		hEnc:            hpack.NewEncoder(&buf),
-		maxStreams:      maxStreams,
-		inTapHandle:     config.InTapHandle,
-		controlBuf:      newRecvBuffer(),
-		fc:              &inFlow{limit: initialConnWindowSize},
-		sendQuotaPool:   newQuotaPool(defaultWindowSize),
-		state:           reachable,
-		writableChan:    make(chan int, 1),
-		shutdownChan:    make(chan struct{}),
-		activeStreams:   make(map[uint32]*Stream),
-		streamSendQuota: defaultWindowSize,
-		stats:           config.StatsHandler,
-		kp:              kp,
-		idle:            time.Now(),
-		kep:             kep,
+		ctx:                   context.Background(),
+		conn:                  conn,
+		remoteAddr:            conn.RemoteAddr(),
+		localAddr:             conn.LocalAddr(),
+		authInfo:              config.AuthInfo,
+		framer:                framer,
+		hBuf:                  &buf,
+		hEnc:                  hpack.NewEncoder(&buf),
+		maxStreams:            maxStreams,
+		inTapHandle:           config.InTapHandle,
+		controlBuf:            newRecvBuffer(),
+		fc:                    &inFlow{limit: uint32(icwz)},
+		sendQuotaPool:         newQuotaPool(defaultWindowSize),
+		state:                 reachable,
+		writableChan:          make(chan int, 1),
+		shutdownChan:          make(chan struct{}),
+		activeStreams:         make(map[uint32]*Stream),
+		streamSendQuota:       defaultWindowSize,
+		stats:                 config.StatsHandler,
+		kp:                    kp,
+		idle:                  time.Now(),
+		kep:                   kep,
+		initialWindowSize:     iwz,
+		initialConnWindowSize: icwz,
```

Comment:
It looks like after their `inFlow`s are created and the connection window updates sent, the `initialConnWindowSize` fields are no longer used by the transport structs, can they be gotten rid of as fields in the structs?